File: Python/Software Development Module - Python/Week3/Eggs.py
"""
Eggs.py
Billy Cussen
25/09/2020
"""

class myEggs:
    def __init__(self, eggs):
        self.eggs = eggs
        self.dozen = 12
        self.boxes = 0
        self.remainder = 0

    def computeBoxesAndRemainder(self):
        self.boxes = int(self.eggs / self.dozen)
        self.remainder = int(self.eggs % self.dozen)

File: Java/Software Development Module - Java/Week3/Eggs.java
/*
Eggs.java
Billy Cussen
08/07/2020
*/

//This Class will compute how many boxes of eggs will be filled based on the amount collected (a dozen/12 eggs per box)

public class Eggs {
    
    //Data Members
    private int eggs;
    private int remainder;
    private int boxes;

    //Constructor
    public Eggs(){
        this.eggs = 0;
        this.remainder = 0;
        this.boxes = 0;
    }

    //Setter
    public void setEggs(int eggs){
        this.eggs = eggs;
    }

    //Compute
    public void computeBoxes(){
        boxes = eggs / 12;
        remainder = eggs % 12;
    }

    //Getters
    public int getBoxes(){
        return boxes;
    }

    public int getRemainder(){
        return remainder;
    }


}